spec/creditsafe/client_spec.rb

```diff
@@ -157,6 +199,27 @@
       it "returns nil" do
         expect(find_company).to be_nil
       end
+
+      it "records a nil payload" do
+        find_company
+        expect(notifications).to match([have_attributes(
+          payload: {
+            request: be_truthy,
+            response: {
+              find_companies_response: include(
+                find_companies_result: include(
+                  messages: {
+                    message: include(
+                      "There are no results matching specified criteria."
```

Comment:
I take it that this huge nested hash is just an unfortunate consequence of using SOAP, and that there isn't a lot that could be done to reduce the size